src/go/configgenerator/route_generator_test.go

```diff
@@ -186,49 +207,45 @@ func TestMakeRouteConfig(t *testing.T) {
 					Rules: []*annotationspb.HttpRule{
 						{
 							Selector: "endpoints.examples.bookstore.Bookstore.Foo",
-							Pattern: &annotationspb.HttpRule_Get{
-								Get: "/v1/{book_name=*}/test/**",
+							Pattern: &annotationspb.HttpRule_Custom{
+								Custom: &annotationspb.CustomHttpPattern{
+									Path: "/v1/{book_name=*}/test/**",
+									Kind: "*",
```

Comment:
This is not intended usage.   Users normally will not specify "*" in custom kind.

Reply:
I think it's supported. https://github.com/googleapis/googleapis/blob/cb7fc620590382a4a2ea6ffdf6f51ae0e77bbbb5/google/api/http.proto#L340

Also we have an integration test for this: https://github.com/GoogleCloudPlatform/esp-v2/blob/96c61ffd01f1c6eb0a90aa96aa8d1e0143f147dd/tests/integration_test/grpc_fallback_test.go#L57

Reply:
I see. Thanks